### PR TITLE
[OC-23]-create-userservice

### PR DESCRIPTION
**Pull request by eminecetin**

### CODE REVIEW

File: src/main/java/com/quiz/service/QuizService.java [hunk 1]
```diff
@@ -1,4 +1,13 @@
 package com.quiz.service;
 
+import com.quiz.model.User;
+
+import java.util.List;
+
 public interface QuizService {
+    String createUser(User user);
```
Comment (eycwave) [inline src/main/java/com/quiz/service/QuizService.java]: why did you write in QuizService 🤔

File: src/main/java/com/quiz/service/UserService.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.quiz.service;
+
+import com.quiz.model.User;
+
+import java.util.List;
+
+public interface UserService {
+    String createUser(User user);
+    List<User> getUsers();
+    User getUser(Long id) throws Exception;
```
Comment (JamshidS) [inline src/main/java/com/quiz/service/UserService.java:10]: `getUserById()`

File: src/main/java/com/quiz/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @Override
+    public String createUser(User user) {
+        try {
+            userRepository.addUser(user);
+            return "User created successfully.";
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "Error creating user.";
```
Comment (JamshidS) [inline src/main/java/com/quiz/service/impl/UserServiceImpl.java:23]:
```suggestion
            return "Error while creating user.";
```

**Review by JamshidS — CHANGES_REQUESTED** [review]

File: src/main/java/com/quiz/service/impl/UserServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        }    }
+
+    @Override
+    public List<User> getUsers() {
+        return userRepository.getAllUsers();
+    }
+
+    @Override
+    public User getUser(Long id) {
+        return userRepository.getUserById(id);
```
Comment (eycwave) [inline src/main/java/com/quiz/service/impl/UserServiceImpl.java:33]: what if there is no user matching the given id number? so i guess you should add if condition.

**Pull request closed without merging**